### PR TITLE
CEDS-1675 Change the case of TXM ids to camelcase

### PR DESCRIPTION
**Pull request by PauloPortugal**
This was the feedback from TxM team.

### CODE REVIEW

File: app/services/audit/AuditService.scala [hunk 1]
```diff
@@ -98,5 +98,5 @@ object AuditTypes extends Enumeration {
 }
 object EventData extends Enumeration {
   type Data = Value
-  val EORI, LRN, MRN, DUCR, DecType, ChangeReason, ChangeDescription, FullName, JobRole, Email, Confirmed, SubmissionResult, Success, Failure = Value
+  val eori, lrn, mrn, ducr, decType, changeReason, changeDescription, fullName, jobRole, email, confirmed, submissionResult, Success, Failure = Value
```
Comment (PatrykRudnicki) [inline app/services/audit/AuditService.scala]: The same question like in movements, what about Success and Failure?

File: test/services/audit/AuditServiceSpec.scala [hunk 2]
```diff
@@ -107,10 +107,10 @@ trait AuditTestSupport extends UnitSpec with ExportsDeclarationBuilder with Scal
   val mockAuditConnector = mock[AuditConnector]
 
   val auditData = Map(
-    EORI.toString -> "eori1",
+    eori.toString -> "eori1",
     LRN.toString -> "lrn1", // FIXME refere to bad thing
     DUCR.toString -> "ducr1",
```
Comment (PatrykRudnicki) [inline test/services/audit/AuditServiceSpec.scala]: Here you're using wrong labels, that contains probably correct value, but this LRN and DUCR are not from EventData

File: app/services/audit/AuditService.scala (hunk 1 above)
Comment (ed0906) [reply app/services/audit/AuditService.scala]: I think they get used as a value, rather than a key in the JSON

Comment (PauloPortugal) [reply app/services/audit/AuditService.scala]: That is a value not a key. TxM wanted the keys to be `camelCase`

File: test/services/audit/AuditServiceSpec.scala (hunk 2 above)
Comment (PauloPortugal) [reply test/services/audit/AuditServiceSpec.scala]: Those are fine, here but I am going to explicitly refer to EvenData.

**Pull request merged**